lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -236,7 +236,7 @@ sub _schedule_iso {
             # Prefer new build jobs over old ones either by cancelling old
             # ones or deprioritizing them (up to a limit)
             try {
-                OpenQA::Events::emit_event(
+                OpenQA::Events::singleton->emit_event(
```

Comment:
It should be `OpenQA::Events->singleton` here. While the function form works too (due to lack of arguments), it is definitely meant to be a class method.

Reply:
Fixed